### PR TITLE
[CI] Disable PR test for `PR/No Test` label

### PR DESCRIPTION
**Pull request by yunjayh**
This commit introduces a new feature to disable pr test if `PR/No test` label is set.

ONE-vscode-DCO-1.0-Signed-off-by: yunjay-hong <yunjay.hong@samsung.com>

---

for: #958 

You can find how theses actions work through #963 (the pr will be closed and the `no_test` branch will be removed after this pr has been merged).

### CODE REVIEW

File: .github/workflows/check-commit.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ on:
 
 jobs:
   check-signed-off-by:
+    if: ${{ ! contains(github.event.pull_request.labels.*.name, 'PR/NO TEST') }}
```
Comment (YongseopKim) [inline .github/workflows/check-commit.yml:9]: For reviews,
- in yml, single quote(') doesn't parse escape characters like `\n` and contains space(' ').

Comment (YongseopKim) [reply .github/workflows/check-commit.yml:9]: (I was not sure so I googled.)

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**